scripts/build.sh

```diff
@@ -0,0 +1,33 @@
+# in-tree build
```

Comment:
probably the scripts in scripts/ are meant to be local scripts and not checked in ?

Reply:
Sounds good. I left it here as the example to build the project. Will remove it in the end.